src/libproc_macro/lib.rs

```diff
@@ -78,9 +78,8 @@ pub struct TokenStream(tokenstream::TokenStream);
 /// Error returned from `TokenStream::from_str`.
 #[stable(feature = "proc_macro_lib", since = "1.15.0")]
 #[derive(Debug)]
-pub struct LexError {
-    _inner: (),
-}
+#[non_exhaustive]
```

Comment:
```
[00:07:40] error: non exhaustive is an experimental feature (see issue #44109)
[00:07:40]   --> /checkout/src/libproc_macro/diagnostic.rs:18:1
[00:07:40]    |
[00:07:40] 18 | #[non_exhaustive]
[00:07:40]    | ^^^^^^^^^^^^^^^^^
[00:07:40]    |
[00:07:40]    = help: add #![feature(non_exhaustive)] to the crate attributes to enable
```

Reply:
I totally thought I added this but I guess not.